alacritty/src/window.rs

```diff
@@ -254,12 +254,19 @@ impl Window {
 
         let class = &window_config.class;
 
+        let fullscreen = if window_config.startup_mode == StartupMode::Fullscreen {
+            Some(Fullscreen::Borderless(None))
+        } else {
+            None
+        };
```

Comment:
The amount of code needed for this and duplication we have here is most unfortunate. But I'm not sure I have a concrete suggestion. In theory we could have a `fullscreen` and `maximized` option on our config, that might help? So then we could just do `with_fullscreen(window_config.fullscreen())`.

Same would have to be done for maximized though unfortunately.